packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -288,6 +300,23 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
     }
   }
 
+  bool canRequestFocus() {
+    bool requestFocus = true;
+    switch (Theme.of(context).platform) {
+      case TargetPlatform.iOS:
+      case TargetPlatform.android:
+      case TargetPlatform.fuchsia:
+        requestFocus = false;
+        break;
+      case TargetPlatform.macOS:
+      case TargetPlatform.linux:
+      case TargetPlatform.windows:
+        break;
+    }
+
+    return widget.requestFocusOnTap ?? requestFocus;
```

Comment:
Although it doesn't matter very much, this would probably be. slightly better with a short circuit return:
```dart
    if (widget.requestFocusOnTap != null) {
      return widget.requestFocusOnTap;
    }
    switch (Theme.of(context).platform) {
      case TargetPlatform.iOS:
      case TargetPlatform.android:
      case TargetPlatform.fuchsia:
        return false
        break;
      case TargetPlatform.macOS:
      case TargetPlatform.linux:
      case TargetPlatform.windows:
        return true;
    }
```